consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/util/ResponsesSummaryUtil.java

```diff
@@ -0,0 +1,138 @@
+     *
+     * @param mcolResultObjects mcol Result list of objects
+     * @return String xml
+     */
+    public String getMcolResultsXml(List<Object> mcolResultObjects) {
+        String mcolClaimDefencesObjectXML = "";
+        try {
+            mcolClaimDefencesObjectXML = xmlToObjectConverter.convertObjectToXml(mcolResultObjects);
+        } catch (JsonProcessingException e) {
+            throw new RuntimeException("Error getting Mcol Results XML", e);
+        }
+        return mcolClaimDefencesObjectXML;
+    }
+
+    /**
+     * convert to Mcol Results XML from List of cmcResultObjects
+     *
+     * @param cmcResults List
+     * @return String xml
+    */
+    public String convertToMcolResultsXml(List<ClaimDefencesResult> cmcResults) {
+        List<String> claimNumbers = new ArrayList<>();
+        StringBuilder claimDefencesObjectXML = new StringBuilder();
+        for (ClaimDefencesResult result : cmcResults) {
+            int defendantId = 1;
+            if (claimNumbers.contains(result.getCaseManRef())) {
+                defendantId = defendantId + 1;
+            } else {
+                claimNumbers.add(result.getCaseManRef());
+            }
+            claimDefencesObjectXML.append(convertToMcolResultsXml(result, defendantId));
+        }
+
+        return claimDefencesObjectXML.toString();
+    }
+
+    public String convertToMcolResultsXml(ClaimDefencesResult result, Integer defendantId) {
+        StringBuilder sbXML = new StringBuilder();
+        DateTimeFormatter zdtFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'Z'",
+                Locale.ENGLISH);
+
+        sbXML.append("<mcolDefenceDetail>")
+                .append("<claimNumber>").append(result.getCaseManRef()).append("</claimNumber>")
+                .append("<defendantResponse defendantId=\"").append(defendantId).append("\" xmlns:ns2=\"http://ws.sdt.moj.gov.uk/2013/mcol/QuerySchema\">")
+                .append("<filedDate>").append(result.getDefendantResponseFiledDate()).append("</filedDate>")
+                .append("<eventCreatedDateOnMcol>").append(
+                        result.getDefendantResponseCreatedDate().format(zdtFormatter)
+                ).append("</eventCreatedDateOnMcol>")
+                .append("<raisedOnMcol>").append(false).append("</raisedOnMcol>")
+                .append("<responseType>").append(result.getResponseType()).append("</responseType>")
+                .append("<defence>").append(result.getDefence()).append("</defence>")
+                .append("</defendantResponse>")
+                .append("</mcolDefenceDetail>");
+
+        return sbXML.toString();
+    }
+
+    public String convertToMcolResultsXml(String inXml) {
+        inXml = inXml.replace("<mcolDefenceDetail>", "<ns3:mcolDefenceDetail  xmlns=\"http://ws.sdt.moj.gov.uk/2013/sdt/targetApp/SubmitQueryRequestSchema\" xmlns:ns3=\"http://ws.sdt.moj.gov.uk/2013/sdt/SubmitQueryResponseSchema\">");
```

Comment:
instead of keeping the xml hardcoded here, please keep in resource file and load it